testCommon/src/testFixtures/java/com/virtuslab/gitmachete/testcommon/TestProcessUtils.java

```diff
@@ -25,14 +25,15 @@ public static String runProcessAndReturnStdout(Path workingDirectory, int timeou
     String stderr = IOUtils.toString(process.getErrorStream(), StandardCharsets.UTF_8);
     String commandRepr = Arrays.toString(command);
 
-    Assert.assertTrue("Stdout of " + commandRepr + " : " + System.lineSeparator() + " " + stdout,
-        !completed || process.exitValue() != 0);
-    Assert.assertTrue("Stderr of " + commandRepr + " : " + System.lineSeparator() + " " + stderr,
-        !completed || process.exitValue() != 0);
-
-    Assert.assertTrue("command ${commandRepr} has not completed within ${timeoutSeconds} seconds;", completed);
+    Assert.assertTrue(
+        "Stdout of ${commandRepr}: ${System.lineSeparator()} ${stdout};" +
+            "\n Stderr of ${commandRepr}: ${System.lineSeparator() ${stderr};" +
+            "\n command ${commandRepr} has not completed within ${timeoutSeconds} seconds;",
```

Comment:
I'd rather put the reason (_...has not completed..._) first, and only then include stdout and stderr